src/components/now-playing-drawer/PlayBar.tsx

```diff
@@ -1,23 +1,161 @@
 import React from 'react'
 
-import { View, StyleSheet, Text, TouchableOpacity } from 'react-native'
+import {
+  StyleSheet,
+  Text,
+  TouchableOpacity,
+  Animated,
+  View,
+  Dimensions
+} from 'react-native'
 
-const styles = StyleSheet.create({
-  container: {}
-})
+import IconPause from 'app/assets/animations/iconPause.json'
+import IconPlay from 'app/assets/animations/iconPlay.json'
+import FavoriteButton from 'app/components/favorite-button'
+import { useThemedStyles } from 'app/hooks/useThemedStyles'
+import { Theme, ThemeColors, useThemeVariant } from 'app/utils/theme'
+
+import AnimatedButtonProvider from '../animated-button/AnimatedButtonProvider'
+
+import { TrackingBar } from './TrackingBar'
+
+const createStyles = (themeColors: ThemeColors) =>
+  StyleSheet.create({
+    root: {
+      width: '100%',
+      height: 46,
+      alignItems: 'center'
+    },
+    container: {
+      height: '100%',
+      width: '100%',
+      paddingLeft: 12,
+      paddingRight: 12,
+      flexDirection: 'row',
+      alignItems: 'center',
+      justifyContent: 'flex-start'
+    },
+    button: {},
+    playIcon: {
+      width: 32,
+      height: 32
+    },
+    icon: {
+      width: 28,
+      height: 28
+    },
+    trackInfo: {
+      height: '100%',
+      flexShrink: 1,
+      flexGrow: 1,
+      alignItems: 'center',
+      flexDirection: 'row'
+    },
+    artwork: {
+      marginLeft: 12,
+      height: 26,
+      width: 26,
+      backgroundColor: 'blue',
+      borderRadius: 2
+    },
+    trackText: {
+      alignItems: 'center',
+      marginLeft: 12,
+      flexDirection: 'row'
+    },
+    title: {
+      fontFamily: 'AvenirNextLTPro-Bold',
+      color: themeColors.neutral,
+      maxWidth: Dimensions.get('window').width / 3,
+      fontSize: 12
+    },
+    separator: {
+      fontFamily: 'AvenirNextLTPro-Bold',
```

Comment:
Can you use our custom `Text` component for these? That way you won't have to set the color or font family

If you really want to do it via styles you can use `...font('bold')` also

Reply:
thx i forgot we had those